MetaMask/metamask-extension
# Updating design tokens package and shadow values with new tokens

## Explanation
Currently, shadows are not standardized across the app

This is a problem because it creates inconsistent UI elevation patterns. The current shadows are also not optimized for darkmode and sometimes don't contrast very well against a dark background making them hard to see.

In order to solve this problem, this pull request updates all instances of shadows in the css to use some combination of the shadow css design tokens

## More Information

* Fixes #15204 
- Going to add documentaiton in another issue so it can be reviewed more thoroughly  https://github.com/MetaMask/metamask-extension/issues/15299

## Screenshots/Screencaps

Because this PR stretches across numerous components I will add the before / after visual changes on each instance of box-shadow code to make it easier for review

## Manual Testing Steps

- Go to the storybook build on this PR
- Hopefully each update has a story which will make it easy to view the shadow changes
- Use the story for each component to see the shadow changes

Alternatively
- Download the build from this PR
- Install
- Open installed extension go onboarding and check out shadows in app

## Pre-Merge Checklist


### + If there are functional changes:
